Tools/WebKitTestRunner/wpe/EventSenderProxyClientWPE.cpp

```diff
@@ -345,60 +345,59 @@ void EventSenderProxyClientWPE::clearTouchPoints()
     m_touchPoints.clear();
 }
 
-void EventSenderProxyClientWPE::touchStart(double time)
+struct EventSenderProxyClientWPE::TouchPointContext {
+    const TouchPoint::State targetState;
+    const std::optional<TouchPoint::State> newState;
+    const WPEEventType eventType;
+    const double time;
+    const WPEModifiers modifiers;
+    WPEView* view;
+};
+
+std::function<bool(EventSenderProxyClientWPE::TouchPoint&)> EventSenderProxyClientWPE::pointProcessor(const TouchPointContext& context)
 {
-    auto* view = WKViewGetView(m_testController.mainWebView()->platformView());
-    for (auto& point : m_touchPoints) {
-        if (point.state != TouchPoint::State::Down)
-            continue;
+    return [=](TouchPoint& point) -> bool {
```

Comment:
It's not very common to use `[=]` captures in WebKit code. IIUC the only things you need to capture is `context`, so you may as well be explicit in the capture list:

```suggestion
    return [context](TouchPoint& point) -> bool {
```

Reply:
nice catch! My argument list kept growing so I moved it into a `context`, and then didn't update the capture type. Thanks